src/settings/Settings.tsx

```diff
@@ -372,8 +372,8 @@ export const SETTINGS: { [setting: string]: ISetting } = {
         controller: new ServerSupportUnstableFeatureController(
             "feature_jump_to_date",
             defaultWatchManager,
-            ["org.matrix.msc3030"],
-            undefined,
+            ["org.matrix.msc3030", "org.matrix.msc3030.stable"],
```

Comment:
All of these features are *AND*ed together when we actually want them to be *OR*ed 🤔

Reply:
Guess it should be extended to string[][] where its an OR of AND groups to offer more flexibility

Reply:
> Guess it should be extended to string[][] where its an OR of AND groups to offer more flexibility

Added support for this 👍

Reply:
Noted this change on the diff in this comment -> https://github.com/matrix-org/matrix-react-sdk/pull/10398#discussion_r1139850466